themes/bootstrap3/templates/Recommend/SideFacets/cluster-list.phtml

```diff
@@ -14,6 +14,7 @@
     'less-label' => $this->transEsc('less') . ' ...',
     'wrapper-class' => false,
     'wrapper-tagname' => 'div',
+    'label-icons' => 'before'
```

Comment:
Would it make more sense to change the default to `before` instead of just explicitly changing the behavior in this one place? Right now, top-facets also uses the truncate mechanism, and it looks strange having some facets with the icon on one side, and other facets with the icon on the other. If there is a preference for "icon on left," then I think perhaps we should just do that everywhere, rather than risk visual inconsistencies.

This also makes me wonder if we need to account for this in right-to-left mode. Should we detect if we're using a RTL language and flip-flop the icon setting in that situation?

Reply:
I think both would make sense. I'll revise.

Reply:
Default changed to 'before'. Looks like no changes are necessary for RTL mode work correctly, unless I'm missing something.